extensions/SemanticMediaWiki/src/MediaWiki/Hooks/ArticleDelete.php

```diff
@@ -19,74 +23,78 @@
 class ArticleDelete {
 
 	/**
-	 * @var Wikipage
+	 * @var WikiPage $wikiPage
 	 */
 	private $wikiPage = null;
 
 	/**
 	 * @since  2.0
 	 *
-	 * @param Wikipage $wikiPage
+	 * @param WikiPage $wikiPage
 	 */
 	public function __construct( &$wikiPage, &$user, &$reason, &$error ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$user". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$reason". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$error". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)